src/format_rules.ml

```diff
@@ -96,4 +96,6 @@ let gen_rules sctx (config : Dune_file.Auto_format.t) ~dir =
         Super_context.add_rule sctx ~mode:Standard ~loc arr;
         add_diff sctx loc alias_formatted ~dir input output));
   Super_context.add_alias_deps sctx alias
-    (Path.Set.singleton (Build_system.Alias.stamp_file alias_formatted))
+    (Path.Set.singleton (Build_system.Alias.stamp_file alias_formatted));
+  Super_context.add_alias_action sctx alias_formatted ~loc:(Some loc) ~locks:[]
```

Comment:
We should use `add_alias_deps ... Path.Set.empty` here. It does the same thing, however it is slightly cheaper.

Reply:
Thanks! Just did so.